freeipa/src/test/java/com/sequenceiq/freeipa/ldap/LdapConfigServiceTest.java

```diff
@@ -1,38 +1,41 @@
 package com.sequenceiq.freeipa.ldap;
 
+import static org.junit.Assert.assertEquals;
+
 import java.util.Optional;
 
 import org.junit.Assert;
```

Comment:
is the assert in the original package?

Reply:
Thanks, I'll correct all assert imports!

Reply:
Fixed.